hrm-domain/hrm-core/case/caseDataAccess.ts

```diff
@@ -220,7 +209,15 @@ const generalizedSearchQueryFunction = <T>(
       return { rows: result, count: totalCount };
     });
 
-    return { cases: rows, count };
+    return {
+      cases: rows.map(r => ({
+        ...r,
+        ...(r.connectedContacts
+          ? { connectedContacts: r.connectedContacts.slice(0, 1) }
```

Comment:
Are we planning on removing the contacts from the case query eventually? If so I imagine the client would need to do a subsequent query to retrieve the contacts specific to the case(s) it's visualizing - when/if needed.

Reply:
[Nevermind :P](https://github.com/techmatters/hrm/pull/877)